src/egon/data/config.py

```diff
@@ -53,6 +53,15 @@ def settings() -> dict[str, dict[str, str]]:
     with open(files[0]) as f:
         return yaml.safe_load(f)
 
+def dataset_boundaries():
+    """Return dataset boundaries from configuration settings"""
+
+    if '--dataset-boundary' in settings()['egon-data']:
+        dataset = settings()['egon-data']['--dataset-boundary']
+    elif '--clip-datasets-to' in settings()['egon-data']:
+        dataset = settings()['egon-data']['--clip-datasets-to']
+
+    return dataset
```

Comment:
Stumbled upon this during merging and just wanted to note that this function isn't necessary. I tried to anticipate this confusion with my [remark][0] that
> The first one is the canonical one so this one will be used when writing to the configuration file or when obtaining settings via the newly added `egon.data.config.settings` function.

When using `egon.data.config.settings` the option will always appear under the key `"--dataset-boundary"`, so I removed "dataset_boundaries" in 7a090f8770d4dd583e5640016427ba74b80b8964. I'll also remove the second option before the merge, as Guido requested.

[0]: https://github.com/openego/eGon-data/pull/159#issuecomment-800513379